x-pack/plugins/security_solution/common/search_strategy/security_solution/cti/index.ts

```diff
@@ -18,7 +18,7 @@ export interface CtiEventEnrichmentRequestOptions extends RequestBasicOptions {
   eventFields: Record<string, unknown>;
 }
 
-export type CtiEnrichment = Record<string, unknown[]>;
+export type CtiEnrichment = Record<string, unknown[]> & { matched: { type: string } };
```

Comment:
We could maybe leverage `ThreatMatchEcs` here if it helps with autocomplete, but this type is intentionally broad so that we are forced to handle null checks on the client; I don't think we can guarantee that this field will always be there.

Reply:
removed